cabal.project

```diff
@@ -1,6 +1,8 @@
 packages: .
 
-source-repository-package
-    type: git
-    location: https://github.com/kadena-io/thyme.git
-    tag: 6ee9fcb026ebdb49b810802a981d166680d867c9
+-- required by pact
+-- these upper bounds are required in order to not break payload validation in chainweb
+constraints: base16-bytestring <1
+constraints: prettyprinter <1.6.1
+constraints: hashable <1.3.1
+constraints: base64-bytestring <1.1
```

Comment:
Unrelated, but needed for tests to pass and pact being compatible with Kadena mainnet.